CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - You can now enable/disable the multi-worker load balancing strategy with an in-editor toggle so that no Uasset files are changed. Select `Enable Multi-Worker` from the drop option from the `Start Deployment` button on the toolbar to use the multi-worker strategy or de-select to use a single worker strategy in the editor. The `Enable Multi-Worker` toggle in World Settings and the command line option  `-OverrideMultiWorker` have been removed as they are now redundant.
 - Enabled packaging the command line arguments when building a mobile client by default.
 - Added settings for the positioning and opacity of the spatial debugger worker region visualisation.
+- You can now filter logs for Local and Cloud deploy separately. Two dropdowns in editor settings now accordingly.
```

Comment:
Could you mention the name of the setting removed and the two added in `quotes` and explain if a previous value was set it will carry forward into both as default?